src/debugger/scalaDebugger.ts

```diff
@@ -49,6 +50,14 @@ function isExtendedScalaRunMain(
   );
 }
 
+function shellOptions(env: Record<string, string>): vscode.ShellExecutionOptions {
+  if (platform() == "win32") {
+    return { executable: "cmd.exe", shellArgs: ["/c"], env }
```

Comment:
does `powershell.exe` works here?

Reply:
powershell doesn't seem to work at all when run as a task and it needs some additional escaping when run manually. And I am also not sure how the executable is called. Anyway, there is no reason to run powershell as we would not be using any shell specific things at all.

Reply:
`powershell.exe` is the thing which I found in the internet when searching for vs code task api examples, but cmd is fine 👍